kdc/httpkadmind.c

```diff
@@ -1178,24 +1284,33 @@ get_keysN(kadmin_request_desc r)
         nsvcs = 1;
     }
 
-    ret = make_keytab(r);
+    if (nsvcs > 4) {
```

Comment:
FIXME comment, make configurable or just support SPNs, and make configurable limit for that.